acceptance-tests/src/test/java/io/blueocean/ath/offline/personalization/FavoritesCardsTest.java

```diff
@@ -100,6 +109,33 @@ public void testMultibranch() throws IOException, GitAPIException {
         dashboardPage.checkFavoriteCardStatus(fullNameMaster, SUCCESS);
         dashboardPage.checkFavoriteCardStatus(fullNameOther, SUCCESS);
 
+        //Check that favorite actions are hidden until pointer is hovering the row
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        Object backgroundColor;
+
+        backgroundColor = js.executeScript(
+            "return getComputedStyle(document.querySelectorAll('.actions-container')[0], ':after').getPropertyValue('background-color')"
+        );
+
+        //check that background color of overlay is  NOT transparent when pointer is NOT hovering the favorite row
+        Assert.assertNotEquals(backgroundColor, "rgba(0, 0, 0, 0)");
+
+        logger.info("move pointer over favorite card actions");
+
+        Actions action = new Actions(driver);
+        WebElement we = driver.findElement(By.xpath("//*[@class=\"pipeline-card success-bg-lite\"]"));
+        action.moveToElement(we).perform();
+
+        //Wait for the hover animation to finish
+        Thread.sleep(150);
```

Comment:
You should be using `wait` here. This is gonna fail at some point probably for a stupid reason. Alway better to poll for the change, rather than sleep

Reply:
will change the thread.sleep to a wait statement, thanks

Reply:
I've seen this pattern used [elsewhere in the ATH](https://github.com/jenkinsci/blueocean-plugin/blob/08b6f1b5adf5ca0f122cc9ddb6a1808915e5727c/acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubEnterpriseCreationPage.java#L50):

`wait.until(ExpectedConditions.visibilityOf(githubEnterpriseCreationButton)).click();`

Maybe something along those lines?

Reply:
Seems that would pass as soon as the animation starts tho, rather than when it's done? 

I find quite frustrating all the uncertainty and "remoteness" of selenium testing.

Reply:
`wait.until` uses `FluentWait` which is basically just polling until something returns not null. Using this pattern anywhere you need to check something removes uncertainty. In this case, you can wait on the fact that the computed css is correct.